### PR TITLE
open jdk 17 build working

### PR DESCRIPTION
**Pull request by githubashutoshsoni**
building is working fine. tested with java 17.0.6.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.7.10'
+    ext.kotlin_version = '1.7.21'
 
     repositories {
         google()
-        mavenCentral()
+        maven { url 'https://jitpack.io' }
```
Comment (ortex) [inline build.gradle:6]: why did you change it?

Comment (githubashutoshsoni) [reply build.gradle:6]: Yeah, I guess this change was not required. I just tested and I'm able to build with mavenCentral() do you want me to revert this?

Comment (ortex) [reply build.gradle:6]: yes, please

Comment (githubashutoshsoni) [reply build.gradle]: Can you check now?

**Comment by ortex** [comment]
Need to fix CI now. You can test it on your fork.

File: app/build.gradle [hunk 2]
```diff
@@ -65,7 +65,6 @@ apply plugin: 'kotlin-android-extensions'
 android {
     compileSdkVersion project.compileSdkVersion.toInteger()
 
-    buildToolsVersion project.buildToolsVersion
```
Comment (ortex) [inline app/build.gradle:68]: why did you delete this one?

Comment (githubashutoshsoni) [reply app/build.gradle:68]: buildToolsVersion is deprecated. and it works just fine without using it. that's why. 

`It is no longer necessary to specify buildToolsVersion if you're using Android plugin for Gradle 3.0.0 or higher, because your project automatically uses a default version of the build tools that the plugin specifies.`

https://stackoverflow.com/questions/69279770/buildtoolsversion-and-compilesdkversion-had-been-deprecated

Comment (ortex) [reply app/build.gradle:68]: got it, thanks

Comment (githubashutoshsoni) [reply app/build.gradle:68]: Yes, That's done. Can we merge this and I was wondering if we could work on CI/CD in a new issue? Since hacktoberfest has started I was wondering if this could be a good candidate for my first commit?

Comment (ortex) [reply app/build.gradle:68]: I prefer to have it all in a single PR. But once it is ready you could close this one and open a new PR, then it will be counting for hacktoberfest.


btw, macos-latest and ubuntu-latest github runner images already have java 17, so you could try to add an additional step after checkout:

```
    - name: Setup Java
      run: echo "JAVA_HOME=$JAVA_HOME_17_X64" >> $GITHUB_ENV

Comment (githubashutoshsoni) [reply app/build.gradle:68]: Alright, Sounds great. I will add this to workflow.
